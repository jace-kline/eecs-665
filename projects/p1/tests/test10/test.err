FATAL [1,1]: Unterminated string literal ignored
