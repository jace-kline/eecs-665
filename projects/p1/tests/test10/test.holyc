"Testing whether the unterminated error works" 123 "
Hello I am Jace "