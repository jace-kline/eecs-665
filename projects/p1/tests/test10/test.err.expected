FATAL [1,52]: Unterminated string literal ignored
FATAL [2,17]: Unterminated string literal ignored